packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -391,6 +420,15 @@ function Preview({ isInspecting, setIsInspecting, zoomLevel, onZoomChanged }: Pr
     e.preventDefault();
   }
 
+  useEffect(() => {
+    if (!focus) {
+      setIsPanning(false);
+      setIsMultiTouching(false);
+      setIsPressing(false);
+    }
+    setIsTouchAreaActive(focus);
+  }, [focus]);
```

Comment:
Feels a bit weird to express this the way you did. It should be enough to have a single effect here instead of state-like variable that uses useState and useEffect under the hood just to use its output in useEffect.

Would be more readable if you just had listeners setup here, or you can modify the helper method to be 
```js
useDocuemntFocusEffect((isDocumentFocused) => {
  if (!isDocumentFocused) {
    // .... 
  }
})```
This way it wouldn't require additional state variable in this component that you never access except in the deps array for this effect here